Bottom: 00ec90e891641151372f615a6fb2938431dd2a0e
Top:    a5ba49c68245d61eaa9038fa596d3092dfdc2a12
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-04-10 15:42:04 -0500

Move common variables to group_vars file

---

diff --git a/ansible-playbook/group_vars/docker-hosts/docker.yml b/ansible-playbook/group_vars/docker-hosts/docker.yml
index aaf9299..9acd909 100644
--- a/ansible-playbook/group_vars/docker-hosts/docker.yml
+++ b/ansible-playbook/group_vars/docker-hosts/docker.yml
@@ -8,3 +8,10 @@ docker_name_prefix: ''
 
 # the user name portion of the docker image name
 docker_registry_username: novatechweb
+
+# the base directory where the docker projects will be checked out
+docker_projects_dir: '/opt/docker/{{ docker_image_tag }}'
+docker_restore_config_base_dir: '/etc/docker_container.conf/{{ docker_image_tag }}'
+
+# restore directories to temporarly store data being restored into docker containers
+docker_backup_dir: '/tmp/docker'
diff --git a/roles/docker-common/defaults/main.yml b/roles/docker-common/defaults/main.yml
index aaf9299..ed97d53 100644
--- a/roles/docker-common/defaults/main.yml
+++ b/roles/docker-common/defaults/main.yml
@@ -1,10 +1 @@
 ---
-
-# The tag of the immage currently being used to build the container
-docker_image_tag: production
-
-# A prefix to seperate containers and images without removing running containers
-docker_name_prefix: ''
-
-# the user name portion of the docker image name
-docker_registry_username: novatechweb
diff --git a/roles/docker-common/vars/main.yml b/roles/docker-common/vars/main.yml
index b4b1526..ed97d53 100644
--- a/roles/docker-common/vars/main.yml
+++ b/roles/docker-common/vars/main.yml
@@ -1,11 +1 @@
 ---
-
-#docker_project_repository: https://github.com/JosephLutz/docker-project.git
-
-# the base directory where the docker projects will be checked out
-docker_projects_dir: '/opt/docker/{{ docker_image_tag }}'
-docker_restore_config_base_dir: '/etc/docker_container.conf/{{ docker_image_tag }}'
-
-# restore directories to temporarly store data being restored into docker containers
-docker_backup_dir: '/tmp/docker'
-docker_restore_dir: '{{ bacula_dest }}{{ docker_backup_dir }}'
